docs/src/content/docs/es/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,288 @@
+
+```js {12,14,18,20}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Tu color de acento preferido. Indigo es el más cercano a los valores predeterminados de Starlight.
+				accent: colors.indigo,
+				// Tu escala de grises preferida. Zinc es el más cercano a los valores predeterminados de Starlight.
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				// Tu fuente de texto preferida. Starlight usa una pila de fuentes del sistema de forma predeterminada.
+				sans: ['"Atkinson Hyperlegible"'],
+				// Tu fuente de código preferida. Starlight usa fuentes monoespaciadas del sistema de forma predeterminada.
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Temas
+
+El tema de color de Starlight se puede controlar anulando sus propiedades personalizadas predeterminadas.
+Estas variables son utilizadas en toda la UI con una gama de tonos grises utilizados para los colores de texto y fondo y un color de acento utilizado para los enlaces y para resaltar los elementos actuales en la navegación.
+
+### Editor de temas de color
+
+Usa los controles deslizantes a continuación para modificar las paletas de colores de acento y gris de Starlight.
+Las áreas de vista previa oscura y clara mostrarán los colores resultantes, y toda la página también se actualizará para obtener una vista previa de sus cambios.
+
+Cuando estés satisfecho con tus cambios, copia el código CSS o Tailwind a continuación y úsalo en tu proyecto.
+
+import ThemeDesigner from '../../../../components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Presets',
+			ocean: 'Ocean',
+			forest: 'Forest',
+			oxide: 'Oxide',
+			nebula: 'Nebula',
+			default: 'Default',
+			random: 'Random',
+		},
+		editor: {
+			accentColor: 'Accent',
+			grayColor: 'Gray',
+			hue: 'Hue',
+			chroma: 'Chroma',
+			pickColor: 'Pick color',
+		},
+		preview: {
+			darkMode: 'Dark mode',
+			lightMode: 'Light mode',
+			bodyText:
+				'Body text is displayed in a gray shade with a high contrast with the background.',
+			linkText: 'Links are colored.',
+			dimText: 'Some text, like the table of contents, has a lower contrast.',
+			inlineCode: 'Inline code has a distinct background.',
+		},
+	}}
+>
+	<Fragment slot="css-docs">
+		Add the following CSS to your project in a [custom CSS
+		file](#custom-css-styles) to apply this theme to your site.
+	</Fragment>
+	<Fragment slot="tailwind-docs">
+		The example [Tailwind config file](#estilando-starlight-con-tailwind) below
+		includes generated `accent` and `gray` color palettes to use in the
+		`theme.extend.colors` configuration object.
+	</Fragment>
+</ThemeDesigner>
```

Comment:
This can all be translated!

The object keys should stay the same (`label`, `ocean`, etc.) but the strings can be translated to Spanish.

The text between the `<Fragment>`s at the ned is regular Markdown documenting how to use the CSS & Tailwind approaches.